coverage-tests/configuration/java_overrides_universal.js

```diff
@@ -18,6 +18,13 @@
     'should handle check of stale element in frame if selector is preserved': {skip: true}, // Not implemented yet
 
     // They are testing a functionality that no longer exists for the SDK
+    'should return actual viewport size': {skip: true},
     'should set viewport size': {skip: true},
     'should set viewport size on edge legacy': {skip: true},
+
+    // Chrome emulator have minor diffs with JS sdk
+    'should not fail if scroll root is stale on android': {config: {branchName: 'universal-java'}},
+    'check window fully on android chrome emulator on mobile page with horizontal scroll': {config: {branchName: 'universal-java'}},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.java3/pullRequest?prid=8925058)
```suggestion
    "check window fully on android chrome emulator on mobile page with horizontal scroll": {config: {branchName: 'universal-java'}},
```